### PR TITLE
chore(i18n): add more languages

### PR DESCRIPTION
**Pull request by Yoiber071**
Add Transifex Working Group languages.

New language codes: it_IT,pt_PT,de_DE

@feanil
@natabene
@igobranco

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Yoiber071!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-export TRANSIFEX_RESOURCE = frontend-app-authn
-transifex_langs = "ar,fr,es_419,zh_CN,it_IT,pt_PT,de_DE,uk,ru,hi"
+export TRANSIFEX_RESOURCE=frontend-app-learning
```
Comment (zainab-amir) [inline Makefile:1]: Why does it say `frontend-app-learning`?

Comment (Yoiber071) [reply Makefile]: Hello, I apologize, I missed the name, it has been corrected.

**Comment by e0d** [comment]
@mphilbrick211 tests running.

**Comment by e0d** [comment]
@Yoiber071 I pulled in the latest changes with a merge commit and reran the tests.  Looks like there are some failures.

**Comment by zainab-amir** [comment]
Lint issue with how the imports are ordered

```
/home/runner/work/frontend-app-authn/frontend-app-authn/src/i18n/index.jsx
Error:    2:1   error  `./messages/fr.json` import should occur after import of `./messages/de_DE.json`      import/order
Error:    3:1   error  `./messages/es_419.json` import should occur after import of `./messages/de_DE.json`  import/order
Error:    4:1   error  `./messages/zh_CN.json` import should occur after import of `./messages/pt_PT.json`   import/order
Error:    5:1   error  `./messages/pt.json` import should occur after import of `./messages/it_IT.json`      import/order
Error:    6:1   error  `./messages/it.json` import should occur after import of `./messages/de_DE.json`      import/order
Error:    7:1   error  `./messages/uk.json` import should occur after import of `./messages/pt_PT.json`      import/order
Error:    9:1   error  `./messages/ru.json` import should occur after import of `./messages/pt_PT.json`      import/order
Error:   10:1   error  `./messages/hi.json` import should occur after import of `./messages/de_DE.json`      import/order
Error:   11:1   error  `./messages/fr_CA.json` import should occur after import of `./messages/de_DE.json`   import/order
Error:   31:26  error  Missing trailing comma                                                                comma-dangle

✖ 10 problems (10 errors, 0 warnings)
```

**Comment by Yoiber071** [comment]
Hi @zainab-amir, Sorry for the late response, I've fixed the lint failures, and is ready to be tested again.

**Comment by mphilbrick211** [comment]
@Yoiber071 Tests have been enabled, though it looks like there's a few failing that will need to be addressed.

**Comment by mphilbrick211** [comment]
> @Yoiber071 Tests have been enabled, though it looks like there's a few failing that will need to be addressed.

Hi @Yoiber071 - friendly ping on this :)

**Comment by mphilbrick211** [comment]
> > @Yoiber071 Tests have been enabled, though it looks like there's a few failing that will need to be addressed.
> 
> Hi @Yoiber071 - friendly ping on this :)

Hi @Yoiber071 - just following up on this.

**Comment by openedx-webhooks** [comment]
@Yoiber071 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-authn&entry.867055334=https://github.com/openedx/frontend-app-authn/pull/747&entry.1484655318=https://github.com/Yoiber071&entry.752974735=2023-02-13+22:23&entry.1917517419=2023-04-13+06:24&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**